src/ai/backend/storage/api/manager.py

```diff
@@ -1163,37 +1163,43 @@ def init_v2_volume_app(service_ctx: ServiceContext) -> web.Application:
     app.router.add_route("GET", "/volumes", handler.get_volumes)
     app.router.add_route("GET", "/volumes/{volume_id}", handler.get_volume)
     app.router.add_route(
-        "GET", "/volumes/{volume_id}/quotas/{scope_type}/{scope_uuid}", handler.get_quota_scope
+        "GET",
+        "/volumes/{volume_id}/quotas/{quota_scope_type}/{quota_scope_uuid}",
+        handler.get_quota_scope,
     )
     app.router.add_route(
-        "POST", "/volumes/{volume_id}/quotas/{scope_type}/{scope_uuid}", handler.create_quota_scope
+        "POST",
+        "/volumes/{volume_id}/quotas/{quota_scope_type}/{quota_scope_uuid}",
+        handler.create_quota_scope,
     )
     app.router.add_route(
-        "PUT", "/volumes/{volume_id}/quotas/{scope_type}/{scope_uuid}", handler.update_quota_scope
+        "PUT",
+        "/volumes/{volume_id}/quotas/{quota_scope_type}/{quota_scope_uuid}",
```

Comment:
In the middle, we also change quotas to quota_scope

Reply:
ok